code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -149,3 +159,21 @@
 		SSmedals.SetScore(BOSS_SCORE, C, 1)
 		SSmedals.SetScore(score_type, C, 1)
 	return TRUE
+
+/datum/action/innate/megafauna_attack
+	name = "Megafauna Attack"
+	icon_icon = 'icons/mob/actions/actions_animal.dmi'
+	button_icon_state = ""
+	var/mob/living/simple_animal/hostile/megafauna/M
+	var/chosen_message = ""
```

Comment:
can this be null?

Reply:
besides the vars and icon_icon dont think i need button icon or name

Reply:
sorry, referring to `var/chosen_message = ""`